### ISSUE TITLE
Dev: WAVE Accessibility - Contrast Ratio Fixes

### ISSUE THREAD

**Issue by ryanfchase** [hackforla/tdm-calculator#2534]
### Dependency
- [ ] #2533 

### Overview
Please fix several text elements whose styles have insufficient Contrast Ratio, as described in https://github.com/hackforla/tdm-calculator/issues/2533

### Action Items
- [ ] Assign yourself to the development issue.
- [ ] Move the development issue from the Prioritized Backlog to In Progress.
- [ ] Fix the following text elements to match specification in design issue
  - [ ] Feedback Form page
    - [ ] modify color for "Project Name" asterisk to be `todo`
    - [ ] "Comment" asterisk
  - [ ] Create New Account page
    - [ ] modify color for "Already have an account?" to be `todo`
  - [ ] Send Password Recovery page
    - [ ] modify color for "New User" to be `todo`
- [ ] Create a PR, if applicable.
- [ ] Create a new slide (or edit one if it already exists) in staging and add the before and after images
- [ ] Once the PR has been approved, use it to close this issue. If there is no PR is associated with this issue, close this issue after lead signoff.

### Resources/Instructions
- Design details: [link to final comment in GitHub issue]
- Design mockups: [link to Ready for Dev section in the related Figma page]

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - [Staging slide, [2534](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g35ba663f517_0_36#slide=id.g35ba663f517_0_36)]
- Release Deck
   - [2025-MM-DD Release deck, slide ISSUE NUMBER]

#### Screenshots

<details><summary>Screenshots: TDM Calculator Feedback Form</summary>
<p>

## Screenshot: TDM Calculator Feedback Form, Overview
![Image](https://github.com/user-attachments/assets/af8aa874-dc5b-499b-9db4-b7f08784236e)

## Screenshot: "Project Name"
![Image](https://github.com/user-attachments/assets/247e801e-96ec-4e57-823c-824c8b13ffed)

## Screenshot: "Comment"
![Image](https://github.com/user-attachments/assets/8e18fbe2-c71f-4269-928e-2394b8667bc5)

</p>
</details> 

<details><summary>Screenshots: Create New Account</summary>
<p>

## Screenshot: Create New Account, Overview
![Image](https://github.com/user-attachments/assets/908c6779-92ff-47ae-9ee8-4b46c96c71e9)

## Screenshot: "Already have an account?"
![Image](https://github.com/user-attachments/assets/8ac0e9a3-380d-4996-8664-41d5cc237850)

</p>
</details> 

<details><summary>Screenshots: Send Password Recovery</summary>
<p>

## Screenshot: Send Password Recovery, Overview
![Image](https://github.com/user-attachments/assets/56554aca-da35-40c5-805f-c0939278dc6f)

## Screenshot: "New User?"
![Image](https://github.com/user-attachments/assets/9358bb8f-4d18-495d-8c68-e6dc6235482a)

</p>
</details>

**Comment by ryanfchase** [comment]
Moved to Icebox while design issue is waiting approval, completion, and review.

**Comment by ryanfchase** [comment]
The design ticket was converted to a dev ticket, so we can close this as a duplicate since the work is completed repeated
- #2533
